scipy/special/add_newdocs.py

```diff
@@ -161,6 +161,62 @@ def add_newdoc(place, name, doc):
 
     """)
 
+
+add_newdoc("scipy.special", "agm",
+    """
+    agm(a, b)
+
+    Compute the arithmetic-geometric mean of `a` and `b`.
+
+    Start with a_0 = a and b_0 = b and iteratively compute::
+
+        a_{n+1} = (a_n + b_n)/2
```

Comment:
TBH, I would just go latex in scipy.special. In some cases it makes sense to list the definition before parameters, and standard notation (iow latex) really seems preferable.